python/sglang/srt/disaggregation/mooncake/conn.py

```diff
@@ -113,6 +113,7 @@ class KVArgsRegisterInfo:
     dst_tp_rank: int
     dst_attn_tp_size: int
     dst_kv_item_len: int
+    page_size: int
```

Comment:
Maybe this would be better?
```suggestion
    dst_page_size: int
```